apps/condo/domains/meter/schema/RegisterMetersService.js

```diff
@@ -0,0 +1,310 @@
+        },
+        {
+            access: true,
+            type: 'input RegisterMetersItemInput { address: String!, addressMeta: RegisterMetersItemAddressMetaInput, accountNumber: String!, accountMeta: RegisterMetersItemAccountMetaInput, meters: [RegisterMetersMeterInput!]! }',
+        },
+        {
+            access: true,
+            type: 'input RegisterMetersInput { dv: Int!, sender: JSON!, organization: OrganizationWhereUniqueInput!, items: [RegisterMetersItemInput!]! }',
+        },
+
+        // OUT
+        {
+            access: true,
+            type: 'type RegisterMetersMeterReadingResultSuccessOutput { id: ID! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersMeterReadingResultErroneousOutput { message: String! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersMeterReadingResultOutput { error: RegisterMetersMeterReadingResultErroneousOutput, data: RegisterMetersMeterReadingResultSuccessOutput }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersMeterReadingOutput { v1: String!, v2: String, v3: String, v4: String, result: RegisterMetersMeterReadingResultOutput }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersMeterResultSuccessOutput { id: ID!, readings: [RegisterMetersMeterReadingOutput!]! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersMeterResultErroneousOutput { message: String! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersMeterResultOutput { error: RegisterMetersMeterResultErroneousOutput, data: RegisterMetersMeterResultSuccessOutput }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersMeterOutput { number: String!, result: RegisterMetersMeterResultOutput! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersItemResultSuccessOutput { propertyId: ID!, meters: [RegisterMetersMeterOutput!]! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersItemResultErroneousOutput { message: String! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersItemResultOutput { error: RegisterMetersItemResultErroneousOutput, data: RegisterMetersItemResultSuccessOutput }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersItemOutput { address: String!, accountNumber: String!, result: RegisterMetersItemResultOutput! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterMetersOutput { items: [RegisterMetersItemOutput!]! }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canRegisterMeters,
+            schema: 'registerMeters(data: RegisterMetersInput!): RegisterMetersOutput',
```

Comment:
Why not MeterReadings? ins not about meters

Reply:
renamed